Update project setup and documentation (#71)
soramitsu/scale-codec-js-library: packages/enum/README.md

Small tagged-union library for TypeScript.

## Features

- **Type narrowing** and **exhaustiveness check** (due to the library following TypeScript's [discriminated union pattern](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#discriminated-unions))

> - Type-safe variants creation
> - Type inference on variants creation

Reviewer: ```suggestion
- Creation of type-safe variants
- Creation of type inference on variants
```

Author: First one is good, but the second is not valid. It means that the method to create a variant is able to infer its typing from the context. How to say it better?

Reviewer: would this work?

```suggestion
- Creation of type-safe variants
- Type inference from variant creation
```

  const { x, y } = event1.content
}
```

## API

> [Link](https://soramitsu.github.io/scale-codec-js-library/api/modules/scale_codec_enum)

Reviewer: ```suggestion
[`@scale-codec/enum` API](https://soramitsu.github.io/scale-codec-js-library/api/enum.html)
```
